files/en-us/web/api/csspropertyrule/index.html

```diff
@@ -0,0 +1,62 @@
+---
+title: CSSPropertyRule
+slug: Web/API/CSSPropertyRule
+tags:
+  - API
+  - CSS
+  - CSS Properties and Values API
+  - CSSPropertyRule
+  - Experimental
+  - Houdini
+  - Interface
+  - Reference
+---
+<div>{{APIRef("CSS Properties and Values API")}}{{SeeCompatTable}}{{draft}}</div>
+
+<p>The <strong><code>CSSPropertyRule</code></strong> interface of the {{domxref('CSS_Properties_and_Values_API','','',' ')}} represents a single CSS {{cssxref("@property")}} rule.</p>
+
+<h2 id="Properties">Properties</h2>
+
+<p><em>Inherits properties from its ancestor {{domxref("CSSRule")}}.</em></p>
+
+<dl>
+	<dt>{{domxref("CSSPropertyRule.inherits")}} {{readonlyinline}}</dt>
+    <dd>Returns the inherit flag of the custom property.</dd>
+    <dt>{{domxref("CSSPropertyRule.initialvalue")}} {{readonlyinline}}</dt>
+    <dd>Returns the initial value of the custom property.</dd>
+    <dt>{{domxref("CSSPropertyRule.name")}} {{readonlyinline}}</dt>
+    <dd>Returns the name of the custom property.</dd>
+    <dt>{{domxref("CSSPropertyRule.syntax")}} {{readonlyinline}}</dt>
+	<dd>Returns the literal syntax of the custom property.</dd>
+</dl>
+
+<h2 id="Methods">Methods</h2>
+
+<p>No specific methods; inherits methods from its ancestor {{("CSSRule")}}.</p>
```

Comment:
As vaguely mentioned in https://github.com/mdn/yari/issues/2168#issuecomment-746941039 this line is probably what's breaking the builder.

Reply:
Just confirming that this is the line that's breaking the builder. I think it should instead be:
```suggestion
<p>No specific methods; inherits methods from its ancestor {{domxref("CSSRule")}}.</p>
```

Reply:
I can confirm that.